### PR TITLE
[API] New directory for jupyter notebooks

### PR DESCRIPTION
**Pull request by spell00**
## Brief summary of changes

New Jupyter notebooks to display how the api works.

### CODE REVIEW

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (xlecours) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:229]: This should link to the one in aces/Loris

Comment (christinerogers) [reply docs/notebooks/LORIS-API_Part1-HTTP.ipynb:229]: @spell00 could you update this today?

**Comment by xlecours** [comment]
@spell00 Remove the outputs of the notebooks before sending to github please.

**Comment by spell00** [comment]
@xlecours I provided the markdowns, but now that I cleared the outputs from the notebooks before sending to Github, the content of the notebooks that I see is very similar to the markdowns. I wonder if it is still necessary? Before clearing the output, I could not see part2 as a notebook in Github, so I understood the necessity of the markdowns, but it seems redundant now. Does it serve another purpose I am not seeing?

**Comment by xlecours** [comment]
> @xlecours I provided the markdowns, but now that I cleared the outputs from the notebooks before sending to Github, the content of the notebooks that I see is very similar to the markdowns. I wonder if it is still necessary? Before clearing the output, I could not see part2 as a notebook in Github, so I understood the necessity of the markdowns, but it seems redundant now. Does it serve another purpose I am not seeing?

Maybe @christinerogers can clarify if the markdown can be useful in our readthedoc. Otherwise, it can be removed so we don't have to make sure it is synchronized whit the actual notebook.

**Comment by spell00** [comment]
@xlecours @christinerogers I think everyone agrees the markdowns are redundant, so I removed them. If anyone thinks of a good reason to include them, it can be added again very easily. I added links to the notebooks in Google colab, so that it can be tried directly in the browser. This way, anyone can test the notebooks even if they don't have Jupyter or Loris installed.

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (xlecours) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:223]: broken link

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (xlecours) [inline docs/notebooks/LORIS_API_Part2_Python-script.ipynb:165]: This is not accurate

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (xlecours) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:223]: broken link

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (xlecours) [inline docs/notebooks/LORIS_API_Part2_Python-script.ipynb:191]: Those functions are a nice addition but I think they should return the response instead of printing. There are information in those responses (body, headers) that would be useful to the users.

**Review by xlecours — CHANGES_REQUESTED** [review]

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (spell00) [inline docs/notebooks/LORIS_API_Part2_Python-script.ipynb:2014]: This block won't be necessary when PR #6775 will be merged. It will add an endpoint listing all Dicoms in a project. It will then be possible to get a complete list of candidates and for which visits there is a Dicom file for a given project. For the moment, it is the only way to get these informations without hardcoding them in the notebook, which is problematic when not executed with raisinbread...

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (spell00) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:223]: This link will be valid when the PR is merged to `aces/main`

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (spell00) [reply docs/notebooks/LORIS-API_Part1-HTTP.ipynb]: I updated this to a link that will not be broken when the notebooks are merged (see my comment below)

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (spell00) [reply docs/notebooks/LORIS_API_Part2_Python-script.ipynb:173]: I changed it, but it looks like Github doesn't track Jupyter notebooks very well...

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (spell00) [reply docs/notebooks/LORIS-API_Part1-HTTP.ipynb]: I updated this to a link that will not be broken when the notebooks are merged (see my comment below)

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (spell00) [reply docs/notebooks/LORIS_API_Part2_Python-script.ipynb:201]: I changed it to return the response. I think you are right, it is more convenient to return the response.

**Comment by spell00** [comment]
Some commands will fail until PR #6780 is merged. I tested using the branch containing the changes and there is no error.

**Review by xlecours — CHANGES_REQUESTED** [review]

**Review by xlecours — APPROVED** [review]

**Comment by christinerogers** [comment]
@driusan requesting final review - this is our last stretch in which @spell00 can readily make changes.  thank you!

**Comment by spell00** [comment]
blocked by #7101

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (spell00) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:7]:
```suggestion
   "#LORIS API Tour 1/2\nThis tutorial is also available as a Google colab notebook so you can run it directly from your browser. To access it, click on the button below: <a href=\"https://colab.research.google.com/github/spell00/Loris/blob/main/docs/notebooks/LORIS-API_Part1-HTTP.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment (spell00) [reply docs/notebooks/LORIS-API_Part1-HTTP.ipynb:7]: Sorry for commenting on my own PR, but I want to point out that this link should be changed to the link I am suggesting here. I am not changing it yet so that it can be tested, because the link I am suggesting will only be available once the PR is merged.

File: docs/notebooks/LORIS-API_Part1-HTTP.ipynb
Comment (spell00) [inline docs/notebooks/LORIS-API_Part1-HTTP.ipynb:7]: The link https://colab.research.google.com/github/aces/Loris/blob/main/docs/notebooks/LORIS-API_Part1-HTTP.ipynb won't work until the PR is merged. For now, for testing purposes, one has to use the notebook in this Pr on this branch : https://colab.research.google.com/github/spell00/Loris/blob/2020-07-20-jupyterApiPart2/docs/notebooks/LORIS-API_Part1-HTTP.ipynb

File: docs/notebooks/LORIS_API_Part2_Python-script.ipynb
Comment (spell00) [inline docs/notebooks/LORIS_API_Part2_Python-script.ipynb:13]: the link https://colab.research.google.com/github/aces/Loris/blob/main/docs/notebooks/LORIS_API_Part2_Python-script.ipynb won't work until the PR is merged. For testing, use https://colab.research.google.com/github/spell00/Loris/blob/2020-07-20-jupyterApiPart2/docs/notebooks/LORIS_API_Part2_Python-script.ipynb

**Comment by spell00** [comment]
@xlecours @driusan It is now passing Travis and ready for final review

**Review by driusan — COMMENTED** [review]
I don't know enough about jupyter notebooks to really review this, I'll merge it based on @xlecours approval..

**Pull request merged**
